TerminalDocs/customize-settings/key-bindings.md

```diff
@@ -0,0 +1,458 @@
+---
+title: Windows Terminal Key Bindings
+description: Learn how to create custom key bindings for Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 05/19/2020
+ms.topic: overview
+ms.service: terminal
+---
+
+# Custom key bindings in Windows Terminal
+
+You can create custom key bindings inside the Windows Terminal that give you control of how you interact with the Terminal using your keyboard.
+
+## Key binding properties
+
+Key bindings can be constructed using the following properties.
+
+### Command
+
+The command executed when the associated keys are pressed.
+
+**Property name:** `command`
+
+**Necessity:** Required
+
+**Accepts:** String
+
+### Keys
+
+Defines the key combinations used to call the command.
+
+**Property name:** `keys`
+
+**Necessity:** Required
+
+**Accepts:** String or array[string]
```

Comment:
As a casual reader, why string or array? And what format does the keybinding have to adhere to? We need to provide a lot more details on this one. Because keychords don't exist yet, and keybindings can only be composed of modifiers and one extra key.

Reply:
Oh. You have it below. Definitely link to it.